### PR TITLE
Add management charge rates to framework definitions

### PR DESCRIPTION
**Pull request by tekin**
Updates the framework definitions to include their management charge rates and wires them into the coda finance report and the data warehouse export for submissions.

### CODE REVIEW

File: app/models/framework/definition/CM_OSG_05_3565.rb [hunk 1]
```diff
@@ -4,6 +4,8 @@ class CM_OSG_05_3565 < Base
       framework_short_name 'CM/OSG/05/3565'
       framework_name       'Laundry Services - Wave 2'
 
+      management_charge_rate BigDecimal('0')
```
Comment (leeky) [inline app/models/framework/definition/CM_OSG_05_3565.rb]: As the generator won't automatically insert this, do we need to add a warning if it isn't set for a framework?

File: app/models/framework.rb [hunk 2]
```diff
@@ -12,4 +12,10 @@ class Framework < ApplicationRecord
     with: /\A40\d{4}\z/,
     message: 'must start with “40” and have four additional numbers, for example: “401234”'
   }
+
+  delegate :management_charge_rate, to: :definition
```
Comment (leeky) [inline app/models/framework.rb]: :+1:

File: app/models/framework/definition/CM_OSG_05_3565.rb (hunk 1 above)
Comment (tekin) [reply app/models/framework/definition/CM_OSG_05_3565.rb]: that's a good shout, although perhaps it's premature to do it now given that we don't plan to onboard any more frameworks for the next while.

Comment (leeky) [reply app/models/framework/definition/CM_OSG_05_3565.rb]: :+1:

**Pull request merged**